addons/web/static/tests/views/calendar/calendar_view.test.js

```diff
@@ -5767,53 +5768,71 @@ test(`calendar with filters and count aggregate`, async () => {
     expect(queryAllTexts(".o_calendar_filter_item span")).toEqual(["partner 1", "2", "partner 2"]);
 });
 
-test.tags("desktop");
```

Comment:
I see that you removed 2 tests for the calendar view. Why is that ? from the look of your changes, it doesn't seems like those features were removed. :thinking: